api/spec/requests/spree/api/line_items_spec.rb

```diff
@@ -24,7 +24,7 @@ module Spree::Api
 
     it "can learn how to create a new line item" do
       get spree.new_api_order_line_item_path(order)
-      expect(json_response["attributes"]).to eq(["quantity", "price", "variant_id"])
+      expect(json_response["attributes"]).to eq(["quantity", "price", "variant_id", "customer_metadata"])
```

Comment:
the specs should be squashed into the same commit the change has been introduced in the controller